common/meta/raft-store/Cargo.toml

```diff
@@ -36,6 +36,8 @@ serde = { version = "1.0.136", features = ["derive"] }
 serde_json = "1.0.78"
 thiserror = "1.0.30"
 tonic = { version = "0.6.2", features = ["transport", "codegen", "prost", "tls-roots", "tls"] }
+ipaddress = "0.1.2"
+bad = "0.1.1"
```

Comment:
what's `bad` is used for?

Reply:
Ok, At first time, I want to directly get the Ok(value) from Result<_,_>, That feature uses the ! (experimental function). I will delete it after fix the `vulnerabilities1 problem. Thanks for reminding me of that.